spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -335,8 +335,13 @@ definitions:
             type: u8
             desc: Status flags
             fields:
-              - 5-7:
+              - 6-7:
                   desc: Reserved
+              - 5-5:
+                  desc: Type of Reported tow
+                  values:
+                      - 0: Time of Measurement
+                      - 1: Other
```

Comment:
I feel like somewhere we need to describe what "Other" could possible be.  I suggest here in the intro paragraph. https://github.com/swift-nav/libsbp/pull/880/files#diff-1669e1e4c34cf7571c33296b08862bedd2a1ce937039283eb73a89b59631fcb7L13

Reply:
Thanks for your feedback Dennis! 🎉 🙏